Dashboard/app/js/templates/navData/monitoring-data.handlebars

```diff
@@ -100,20 +100,17 @@
             <!-- TABLE HEADER-->
             <thead>
                 <tr>
-                    <th class="noArrows">{{t _id}}</th>
-                    <th class="noArrows">{{t _question}}</th>
-                    <th class="noArrows">{{t _answer}}</th>
+                    <th class="noArrows" style="text-align:left;font-weight:bold;">{{t _question}}</th>
```

Comment:
Please remove the styling. Its not really needed as the CSS already has this style defined for `noArrows` class